tsconfig.json

```diff
@@ -4,17 +4,17 @@
     "allowJs": false,
     "allowSyntheticDefaultImports": true,
     "declaration": true,
+    "declarationDir": "lib",
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
     "lib": ["es5", "es6", "dom"],
     "moduleResolution": "node",
     "noUnusedLocals": true,
-    "outDir": "lib",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es6"
+    "target": "es2018"
```

Comment:
Target is being bumped to remove `tslib` usage in running rollup build with `@rollup/plugin-typescript`. tslib was used for `__rest` utility which is the spread object functionality that comes with `es2018`. `tslib` is being grepped for as a build test